piptools/utils.py

```diff
@@ -221,10 +203,10 @@ def dedup(iterable):
 def name_from_req(req):
     """Get the name of the requirement"""
     if hasattr(req, 'project_name'):
-        # pip 8.1.1 or below, using pkg_resources
```

Comment:
same: I believe this will be stable now that we use vendored pip

Reply:
See comment above: same applies.

Reply:
ok